packages/compass-connections/src/stores/connections-store.tsx

```diff
@@ -360,7 +366,11 @@ export function useConnections({
         Pick<ConnectionInfo, 'id'>
```

Comment:
Also looking closer at this, I'm pretty sure we never actually pass the partial here, so maybe this special case type is not even needed. I copied this from previous implementation without digging too deep

Reply:
actually we do pass partial in `oidcUpdateSecrets`

Reply:
Ahhh, true, missed that! Hmmm, that method is internal to this hook logic though and we do have the whole connection info object passed to it so we probably can do the merge there ourselves? Totally no strong preference though, mostly saying this because I saw your discussion above about this whole partial / full info type and maybe if we want to remove the ambiguity, we can just refactor away the use-case for this partial type